### ISSUE TITLE
Not getting camera feed

### ISSUE THREAD

**Issue by AngelBlaizeC** [qH0sT/AndroSpy#79]
Not sure if its a bug or anything but the rat is no longer getting camera feed? what I mean is it was working perfectly for few days.. then for some reason I'm getting no buffer from the camera feed. 

![image](https://user-images.githubusercontent.com/78591190/114598989-1df4b980-9cc5-11eb-8ecb-3de552627eb8.png)

I can press stop / start but no more feed.

**Comment by 1337C0D3RS** [comment]
set quality to 10% and try

**Comment by qH0sT** [comment]
@AngelBlaizeC 
be sure that connection is avaible-live between client and server

**Comment by AngelBlaizeC** [comment]
Yes both are live. I can connect / download files. but when viewing the camera is no longer possible.

**Comment by qH0sT** [comment]
> Yes both are live. I can connect / download files. but when viewing the camera is no longer possible.

Have you intervened to source code? + turn on the screen of phone and try again.

**Comment by AngelBlaizeC** [comment]
nope i am using the original source code. it works again when i reinstall the app.

**Comment by qH0sT** [comment]
> nope i am using the original source code. it works again when i reinstall the app.

I think this is problem of miss permissions

**Comment by AngelBlaizeC** [comment]
i encountered the issue again.

the cam was working for few seconds. but the androspy server crash.. when i try to connect again. the client no longer feed the camera.

**Comment by qH0sT** [comment]
> i encountered the issue again.
> 
> the cam was working for few seconds. but the androspy server crash.. when i try to connect again. the client no longer feed the camera.

I have released new update, can you try?

**Comment by 1337C0D3RS** [comment]
same issue here.... camera works sometimes and some times it doesn't...

please look into it..

**Comment by 1337C0D3RS** [comment]
@qH0sT  please fix camera issue also... it won't work most of the times... (sometimes it magically works without doing anything)
